tutorials/migrating/upgrading_to_godot_4.rst

```diff
@@ -460,6 +488,25 @@ table to find its new name.
 - The ``Engine.editor_hint`` property was removed in favor of the
   ``Engine.is_editor_hint()`` *method*. This is because it's read-only, and
   properties in Godot are not used for read-only values.
+- InputEventWithModifiers's ``alt()`` is now ``alt_pressed()``.
+- InputEventWithModifiers's ``command()`` is now ``command_pressed()``.
+- InputEventWithModifiers's ``control()`` is now ``ctrl_pressed()``.
+- WorldMarginShape2D's ``d()`` is now ``distance()``.
+- AudioServer's ``device()`` is now ``output_device()``.
+- InputEventMouseButton's ``doubleclick()`` is now ``double_click()``.
+- Node's ``filename()`` is now ``scene_file_path()``.
+- BaseButton's ``group()`` is now ``button_group()``.
+- InputEventWithModifiers's ``meta()`` is now ``meta_pressed()``.
+- PathFollow2D's ``rotate()`` is now ``rotates()``.
+- Theme's ``off()`` is now ``unchecked()``.
+- Theme's ``ofs()`` is now ``offset()``.
+- Theme's ``on()`` is now ``checked()``.
+- PathFollow2D's ``offset()`` is now ``progress()``.
+- PathFollow3D's ``offset()`` is now ``progress()``.
```

Comment:
Both of these are already listed above, make sure you read through the existing ones